spec/API_specification/elementwise_functions.md

```diff
@@ -968,7 +968,7 @@ For floating-point operands,
 (function-logical_and)=
 ### logical_and(x1, x2, /)
 
-Computes the logical AND for each element `x1_i` of the input array `x1` with the respective element `x2_i` of the input array `x2`. Zeros are considered the equivalent of `False`, while non-zeros are considered the equivalent of `True`.
+Computes the logical AND for each element `x1_i` of the input array `x1` with the respective element `x2_i` of the input array `x2`. Only boolean data types are required, but if non-boolean data types are implemented, zeros are considered the equivalent of `False`, while non-zeros are considered the equivalent of `True`.
```

Comment:
May be better to move this sentence to a note, rather than as the main description of the API description.